interface.go

```diff
@@ -37,6 +37,8 @@ type Session interface {
 	RemoteAddr() net.Addr
 	// Close closes the connection. The error will be sent to the remote peer in a CONNECTION_CLOSE frame. An error value of nil is allowed and will cause a normal PeerGoingAway to be sent.
 	Close(error) error
+	// WaitUntilClosed() blocks until the session is closed.
```

Comment:
Please add a comment that this signature might change (e.g. we might want to add an error down the road).